src/js/views/patients/sidebar/patient/patient-sidebar.scss

```diff
@@ -15,3 +15,9 @@
   font-size: 24px;
   margin-bottom: 8px;
 }
+
+.worklist-patient-sidebar__widgets {
+  .spinner {
+    margin-top: 60px;
+  }
+}
```

Comment:
Added `60px` of top margin for the spinner. Matches the margin used for the activity section of the action/flow sidebar: https://github.com/RoundingWell/care-ops-frontend/blob/develop/src/sass/modules/sidebar.scss#L57.

Let me know if we want to use a different margin for this. I put a video of the sidebar with the spinner in the pull request description so you can see what it looks like.